README.md

```diff
@@ -1 +1,51 @@
-# Hello-Microverse-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse-Project
+
+> This is my first Microverse project.
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+    -Web browser
+    -Internet connection
+
+### Install & Usage
+    -Download the files from the repository and view the page locally, very simple!
+
+
+
+
+
+## Author
+
+👤 **Inaki Eskauriatza**
+
+- GitHub: [@inakiesk](https://github.com/inakiesk)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/i%C3%B1aki-eskauriatza-b82684241?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3B1VEaqHfJTmWAhjqdjRvumA%3D%3D)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thank you Microverse, Youtube, Wikipedia and W3Schools.
+
+
```

Comment:
- [OPTIONAL] I can see you are using the Microverse template, good job 👏 I suggest adding the ` License` section and creating the [MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md) file to make a professional README file 👌.

![image](https://user-images.githubusercontent.com/45231276/176344829-f9b87291-29a1-46e8-b74c-58f656dc17ea.png)


> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs.